include/examples/quadruped_3dof_limb.h

```diff
@@ -0,0 +1,323 @@
+/**
+ * @file quadruped_3dof_limb.h
+ * @author Kodlab - J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief A LimbBase derived class example of the common quadrupedal serial limb
+ * configuration with a point toe
```

Comment:
Provide some an example of a robot with this limb (mit mini cheetah)

Reply:
- [ ] Make working example